api/routes/config.py

```diff
@@ -15,12 +17,14 @@ async def export_config(
     server_name: str
```

Comment:
would be a good idea to validate this input

Reply:
It is validated, the function goes through all of the user's server configs; if not config with the given name is found, then we return 404.